packages/client/src/components/ActivityTable.spec.js

```diff
@@ -0,0 +1,39 @@
+import {
+  beforeEach, describe, it, expect,
+} from 'vitest';
+import Vuex from 'vuex';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import ActivityTable from '@/components/ActivityTable.vue';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+let wrapper;
+const stubs = ['b-table', 'b-icon', 'b-iconstack'];
+
+describe('ActivityTable.vue', () => {
+  describe('when view is loaded', () => {
+    beforeEach(() => {
+      wrapper = shallowMount(ActivityTable, {
```

Comment:
I dont _think_ that mocks or store are needed for this file? maybe?

Reply:
Agreed!